aea/configurations/constants.py

```diff
@@ -26,6 +26,9 @@
 _ETHEREUM_IDENTIFIER = "ethereum"
 _COSMOS_IDENTIFIER = "cosmos"
 SIGNING_PROTOCOL = "open_aea/signing:latest"
+SIGNING_PROTOCOL_WITH_HASH = (
+    "open_aea/signing:latest:QmTCQKD2iSjBUC3QCQRc7ZyNbnnrvoyw7EcQdJa7mQYLUW"
+)
```

Comment:
Since we now specify the hash, let's also specify the version (as it is no longer generic "latest")